xbmc/playlists/PlayListXSPF.cpp

```diff
@@ -0,0 +1,188 @@
+/*
+ *  Copyright (C) 2018 Tyler Szabo
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "PlayListXSPF.h"
+
+#include "utils/log.h"
+#include "utils/URIUtils.h"
+#include "utils/XBMCTinyXML.h"
+#include "URL.h"
+
+using namespace PLAYLIST;
+
+/*
+
+Specification: http://www.xspf.org/specs/
+
+Example:
+
+<?xml version="1.0" encoding="UTF-8"?>
+<playlist xmlns="http://xspf.org/ns/0/" version="1">
+	<title>KEXP</title>
+	<trackList>
+		<track>
+			<location>http://live-aacplus-64.kexp.org/kexp64.aac</location>
+			<title>KEXP.ORG 90.3FM - where the music matters</title>
+      <info>https://kexp.org/</info>
+      <image>https://www.kexp.org/static/assets/img/logo-white.svg</image>
+		</track>
+    <track><location>Relative/Path/To.mp3</location></track>
+    <track><location>file:///music/song_1.ogg</location></track>
+    <track><location>file:///music/song_2.flac</location></track>
+    <track><location>file:///music/song_3.mp3</location></track>
+	</trackList>
+</playlist>
```

Comment:
The indentation in the above block looks odd.

Reply:
Oh shoot, it's the "tabs" I thought I replaced them with spaces. Thanks for catching this; I'll do a grep-pass to see if I did this in any other files.